x-pack/metricbeat/module/oracle/connection.go

```diff
@@ -42,7 +42,7 @@ func HostParser(mod mb.Module, rawURL string) (mb.HostData, error) {
 	}
 
 	if params.Password.Secret() == "" {
-		params.StandaloneConnection = true
+		params.StandaloneConnection = godror.Bool(true)
```

Comment:
API changes. StandaloneConnection now expects a type returned by Bool instead of Go's builtin type bool